### ISSUE TITLE
Clarification, backwards support to status-update instances (aka Mastodon, Pleroma, Misskey etc)

### ISSUE THREAD

**Issue by guysoft** [yuforium/activity-pub-docs#3]
Hey,
Just to confirm, I want to have an instance that both implements both Community Federation and that can interact with the already Mastodon, Pleroma and Misskey instances.

Would theses roughly translate:
``Person`` -> ``Forum``
``Status`` -> ``Topic``
``A list of Persons`` -> ``Community``
?

**Comment by cpmoser** [comment]
@guysoft yes on the first one, a `Forum` is effectively a multi person `Person` with permissions for specific users to post to the forum's outbox (or delete or edit messages).

As far a `Topic` it's not a status, I would consider it the subject matter of what's being talked about.  Maybe there's a better word to describe because I know that `Topic` can be a forum's equivalent to `Status`.  We could introduce something like `Post` (equal to `Status`) or `Thread` which would be a post and a collection of replies.

`Community` I would define as more ephemeral, I think it's a good word to describe the larger federation model of sharing the topic (subject matter) of something.  It would represent all of the connections between forum instances which could be very broad and far reaching.

So what I would propose maybe is a terminology change, something like this:
`Person` -> `Forum`
`Status` -> `Post`, and `Thread` to represent the post + replies

`Topic` would be something new, I would see it as a conduit for forum instances (and potentially individual users too) to share content based on content.

**Comment by guysoft** [comment]
Not sure I fully understand.  Ill try and implement something and ask for input when its doing something "

**Comment by cpmoser** [comment]
I should probably create a diagram for this, that would probably help.

**Comment by guysoft** [comment]
Ok, I tested setting ``Forum`` instead of ``Person`` on Pleroma and it does see the profile correctly.

**Comment by guysoft** [comment]
Implemented here: https://github.com/guysoft/fedigroup

**Comment by guysoft** [comment]
Got stuck, thought I might update here that it made me open this issue:
https://github.com/mastodon/mastodon/issues/18741

**Comment by guysoft** [comment]
Hey, 
so good news, I managed to get Mastodon and Pleroma to show up user profiles.

One of the things that got me stuck was that Mastodon will not show up "Forum" actor types, but will show Person in searches:

This works:
```json
{
  "@context": [
    "https://www.w3.org/ns/activitystreams",
    "https://w3id.org/security/v1"
  ],
  "id": "https://pleroma.gnethomelinux.com/group/aaa2",
  "inbox": "https://pleroma.gnethomelinux.com/inbox",
  "preferredUsername": "aaa2",
  "publicKey": {
    "id": "https://pleroma.gnethomelinux.com/group/aaa2#main-key",
    "owner": "https://pleroma.gnethomelinux.com/group/aaa2",
    "publicKeyPem": "BOOP"
  },
  "type": "Person"
}
```

This does not:
```json
{
  "@context": [
    "https://www.w3.org/ns/activitystreams",
    "https://w3id.org/security/v1"
  ],
  "id": "https://pleroma.gnethomelinux.com/group/aaa2",
  "inbox": "https://pleroma.gnethomelinux.com/inbox",
  "preferredUsername": "aaa2",
  "publicKey": {
    "id": "https://pleroma.gnethomelinux.com/group/aaa2#main-key",
    "owner": "https://pleroma.gnethomelinux.com/group/aaa2",
    "publicKeyPem": "BOOP"
  },
  "type": "Forum"
}
```
